src/panels/config/logs/dialog-system-log-detail.ts

```diff
@@ -69,13 +68,9 @@ class DialogSystemLogDetail extends LitElement {
     const integration = getLoggedErrorIntegration(item);
 
     return html`
-      <ha-paper-dialog
-        with-backdrop
-        opened
-        @opened-changed="${this._openedChanged}"
-      >
-        <div class="heading">
-          <h2>
+      <ha-dialog open @closed=${this.closeDialog} hideActions heading=${true}>
+        <div slot="heading" class="heading">
+          <h2 class="title">
```

Comment:
use `createCloseHeading` https://github.com/home-assistant/frontend/blob/dev/src/panels/config/users/dialog-user-detail.ts#L65-L66

Reply:
Can't do that with the copy icon being there as well?

Reply:
So I should actually expand the stuff Github hides for me to see what the rest of the code is... 🤦